libs/common/src/sql-db-connection/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Base image
-FROM mcr.microsoft.com/mssql/server
+FROM mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
Is there any reason to not use the more recent `mcr.microsoft.com/mssql/server:2022-CU13-ubuntu-22.04` image?